BUILDING.md

```diff
@@ -135,6 +135,15 @@ $ make test
 At this point you are ready to make code changes and re-run the tests!
 Optionally, continue below.
 
+If you wish to run tests on specific systems, say ``fips``:
```

Comment:
Nit: FIPS is not a system. (Is it more correct to call it a "configuration"?)

Reply:
This file has been removed as per your suggestion.